nodes/widgets/ui_radio_group.js

```diff
@@ -24,7 +24,7 @@ module.exports = function (RED) {
                     // dynamically set "columns" property
                     statestore.set(group.getBase(), node, msg, 'columns', updates.columns)
                 }
-                if (updates?.options) {
+                if (updates.options) {
```

Comment:
why are you making this change?

Reply:
It didn't throw any errors when initialising. So I removed because of that. But you're correct better to handle it. 

I will add it back

Reply:
It's perfectly reasonable for a message to be sent without `msg._update` though, so the `?` is a guard against that and should be kept.

Reply:
Reverted the removal of null checker

Reply:
> It's perfectly reasonable for a message to be sent without `msg.ui_update` though, so the `?` is a guard against that and should be kept.

Understood the scenario. I added it back. Thanks for the clarification